.node-version

```diff
@@ -1 +1 @@
-20.10.0
+20.11.0
```

Comment:
Awaiting [node-build release](https://github.com/nodenv/node-build/tree/master/share/node-build) so we can install this locally via `nodenv`

Reply:
Awaiting merge and deployment: https://github.com/nodenv/node-build/pull/847